src/runtime/storage/storage-provider-factory.ts

```diff
@@ -92,7 +92,7 @@ export class StorageProviderFactory {
   }
 
   // For testing
-  shutdown() {
-    Object.values(this._storageInstances).map(item => item.storage.shutdown());
+  async shutdown(): Promise<void> {
+    await Promise.all(Object.values(this._storageInstances).map(item => item.storage.shutdown()));
```

Comment:
Same here.

Reply:
Here the contract should be clearer, even for a test method.  If we want to make this fire-and-forget it should probably be documented as such, and marked as returning void.  I think this is fine as-is; but would be open to revisiting if you feel strongly about it.

Reply:
No, I'm fine with having the awaits.